test/db-sync-progress.js

```diff
@@ -209,7 +209,38 @@ test('sync progress: 200 entries', function (t) {
         t.equals(sofarA, 200)
         t.equals(totalB, 200)
         t.equals(totalB, 200)
+        var stats = db1.getDatabaseStatus()
+        t.same(stats.length, 2)
+        t.same(stats[0].sofar, stats[0].total)
+        t.same(stats[1].sofar, stats[1].total)
```

Comment:
```suggestion
        t.same(stats.length, 2, 'database status feed entries ok')
        t.same(stats[0].sofar, stats[0].total, 'database status feed 1 sofar=total ok')
        t.same(stats[1].sofar, stats[1].total, 'database status feed 2 sofar=total ok')
```

If this cool with you! I'm trying to get in the habit of writing test messages, so travis etc debugging is a bit easier.